packages/adapters/fastify/web/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@redwoodjs/fastify-web",
+  "version": "6.0.7",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/adapters/fastify/web"
+  },
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
```

```suggestion
      "default": "./dist/web.js"
```
(after other changes, as commented on the web.ts file)

---

Done in https://github.com/redwoodjs/redwood/pull/9884/commits/e477c530410bac2703a241b8de7bd1d12b430d09